06_neighborhoods/Transportation/transit_cost_county.R

```diff
@@ -0,0 +1,189 @@
+###################################################################
+
+# Transit Metrics: Transportation Costs
+# Tina Chelidze 
+# 2023-2024 Mobility Metrics update
+# Process (for overall, not subgroup, data):
+# 1. For County: Import and reshape data
+# 2. QC checks
+# 3. Add Data Quality marker
+# 4. Export final files
+
+###################################################################
+
+
+# Libraries you'll need
+library(tidyr)
+library(dplyr)
+library(readr)
+library(ggplot2)
+library(tidyverse)
+library(purrr)
+
+# SUMMARY-LEVEL VALUES
+# Raw data pulled from https://htaindex.cnt.org/download/
+# the Housing and Transportation (H+T) Affordability Index from the Center for Neighborhood Technology (CNT)
+
+###################################################################
+
+# 1. Import all the files (and/or combine into one file) with only the relevant variables and years
+
+# FOR COUNTIES - import the raw data files
+# Unlike for Places and Tracts, the CNT website allows you to download raw data for all counties at once
+# so we just import the all-county file for the two available years; 2015 and 2019
+# 2015
+# raw data lives in the Urban Box folder (replace USERNAME)
+transport_county_2015 <- read_csv("C:/Users/USERNAME/Box/Lab/Projects/Gates Upward Mobility Framework/Outreach and Tools/Data/Metrics_2024_round/Transportation/htaindex2015_data_counties.csv")
```

Comment:
The automation did not work because of the ZIP nature of each file and the need for concatenation (maybe someone can try again in the future), but data download description will be added to the documentation